docs/source/best-practices/checkpointing.rst

```diff
@@ -2,21 +2,21 @@
 Checkpoint/Restart Discovery Jobs
 *****************************************
 
-The complexity of HPC systems may introduce unpredictable behaviors and may result in job failures due to hardware or software. Applying fault tolerance techniques to your HPC workflows allows your jobs to become more resilient to crashes, partition time limits and hardware failures.
+The complexity of HPC systems may introduce unpredictable behaviors and result in job failures due to hardware or software. Applying fault tolerance techniques to your HPC workflows makes your jobs more resilient to crashes, partition time limits, and hardware failures.
 
 Checkpointing will allow you to:
 
  * Create resilient workflows in the existence of faults.
  * Overcome most scheduler resource time limitations.
- * Implement an early error detection approach by inspecting intermediate results.  
+ * Implement an early error detection approach by inspecting intermediate results.
 
 The Checkpointing technique
 ================================
 
 Checkpointing is a fault tolerance technique designed to overcome the “fail-stop” failure type (interruption of the execution of a job). It is based on the BER technique (Backward error recovery or Rollback-recovery algorithm):
```

Comment:
For clarity, can we change to something like: "Checkpointing is a fault tolerance technique based on the backward error recovery (BER) technique and is designed to overcome the “fail-stop” failure type (interruption of the execution of a job)."